lib/ticket-types.js

```diff
@@ -2,7 +2,7 @@
 
 function ticketTypes (args, callback) {
   var ticketTypes = [
-    {name: 'ninja', title: 'Ninja', tooltip: 'Select Ninja to create tickets for attendees under 13 or over 13.'},
+    {name: 'ninja', title: 'Youth', tooltip: 'Select Youth to create tickets for attendees under 18.'},
```

Comment:
I wasn't quite sure if this would mess with what is currently in the database, but it seems to be okay. Will update the other PR too :)